src/bite.v1.js

```diff
@@ -21,9 +22,41 @@
         request.send()
     }
 
+    function determineSession() {
+        const cookies = document.cookie
+            .split('; ')
+            .map(cookie => cookie.split('='))
+            .reduce((mapping, [key, value]) => ({
+                ...mapping,
+                [key]: value,
+            }), {})
+
+        if (cookies['barnebys_session']) {
+            return cookies['barnebys_session']
+        }
+
+        const currentLocation = new URL(window.location)
+        const searchParams = new URLSearchParams(currentLocation.search)
```

Comment:
URL and URLSearchParams needs to be polyfilled in order to support older browsers

Reply:
It's not too necessary though, the values can be extracted simply using string methods/regex as well

Reply:
Replaced it with as custom function as it felt overkill to add a polyfill